### PR TITLE
Add base_dir flag to prepare_bigquery

### PR DESCRIPTION
**Pull request by sethtroisi**
This is approximately 4x faster (cProfile now says 90% of time is in sgf.parse)

### CODE REVIEW

File: oneoffs/prepare_bigquery.py [hunk 1]
```diff
@@ -21,65 +21,81 @@
...
-            for sgf_name in sgf_names:
-                game_data, move_data = extract_data(sgf_name)
-                game_f.write(json.dumps(game_data))
-                game_f.write('\n')
-                for move_datum in move_data:
-                    move_f.write(json.dumps(move_datum))
-                    move_f.write('\n')
-                print('processed {}'.format(sgf_name))
+BASE_GS_DIR = 'gs://{}'.format(BUCKET_NAME)
+HOLDOUT_PATH = '{}/data/holdout/{}'
```
Comment (amj) [inline oneoffs/prepare_bigquery.py]: should these be os.path.join?

Comment (sethtroisi) [reply oneoffs/prepare_bigquery.py]: Done

**Comment by brilee** [comment]
Wait, i'm curious - what was the exact change that made it 4x faster?

**Comment by sethtroisi** [comment]
I'm now getting 25games/s which is 20x what I get right now.
4x from not reading SGFs from the bucket
5x from [this patch to sgf](https://github.com/jtauber/sgf/compare/master...sethtroisi:master?expand=1#diff-84c2feb09a1cea380f8d9c0f01a71568R184)

the last profile I ran had JSON + disk reading being 60% of the execution time

**Comment by brilee** [comment]
👏 that's pretty awesome.

**Comment by amj** [comment]
LGTM ok2merge

**Pull request merged**